### PR TITLE
feat(visualizer, chat): add redux logic for the custom visualizer connector ( Issue #1270)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

Add redux actions, selectors and epic. 
Also, change UI for the attachments according to the latest  figma design.
Update README for the Chat Visualizer Connector.

Issues:

- Issue #1270 

**UI changes**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1400.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1400.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1400.nightly-test.deltixhub.io
>E2E tests status: success

File: libs/shared/src/types/visualizer-connector.ts [hunk 1]
```diff
@@ -12,3 +12,15 @@ export interface VisualizerConnectorOptions {
 }
 
 export type VisualizerConnectorRequest = DialLibRequest;
+
+export interface CustomVisualizerDataLayout extends Record<string, unknown> {
```
Comment (IlyaBondar) [inline libs/shared/src/types/visualizer-connector.ts]: Why do we have `extends Record<string, unknown>` in both classes? If we start to support some new properties we will add them explicitly

**Review by Alexander-Kezik — DISMISSED** [review]

File: libs/shared/src/types/visualizer-connector.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply libs/shared/src/types/visualizer-connector.ts]: You are right!
Changed as suggested.

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1400.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**